docs/buildah-build.1.md

```diff
@@ -389,7 +389,8 @@ Valid _mode_ values are:
 - **none**: no networking;
 - **host**: use the host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure;
 - **ns:**_path_: path to a network namespace to join;
-- `private`: create a new namespace for the container (default)
+- **private**: create a new namespace for the container (default)
+- **\<network name|ID\>**: Join the network with the given name or ID. Only supported for rootful users.
```

Comment:
Example for this would be nice, is this `--network <networkname>`.

Reply:
Do you mean in the description or under the example section at the bottom of the page?

Reply:
Yes but its not a blocker for me. I figured cli from `tests`